### DISCUSSION TITLE
Use Vercel logo into our website as our sponsor?

### DISCUSSION THREAD

**Discussion by warengonzaga** [vercel/community#5451] [Help] [answered]
### Summary

Hi there, I'm Waren, co-founder of [OSSPH.ORG](https://ossph.org), and we are reaching out here to see if we can use the logo of Vercel on our website to show that Vercel sponsors us. Are we allowed to do that? Any guidelines or direction is much appreciated.

### Example

https://ossph.org

### Steps to Reproduce

N/A

**Comment by xonx4l** [comment]
If you are sponsored then yes you are allowed to use the logo .

**Reply by warengonzaga** [reply]
Any reference for that? Just making sure we are really allowed to use the logo/branding.

**Reply by ryderhorne** [reply, marked as answer]
Their brand usage terms can be found here: [https://vercel.com/design/brands#usage](https://vercel.com/design/brands#usage)

As long as you have written permission that you are allowed to use their logo to indicate you're sponsored by them, you should be fine.

**Reply by warengonzaga** [reply]
Hey @ryderhorne unfortunately they didn't responded to my email. :/
